package/MDAnalysis/coordinates/DCD.py

```diff
@@ -578,6 +579,7 @@ def timeseries(self, asel=None, start=None, stop=None, step=None, skip=None,
         # XXX needs to be implemented
         return self._read_timeseries(atom_numbers, start, stop, step, format)
 
+    @deprecate(message="This method will be removed in 0.17")
```

Comment:
All the deprecations can go in now. 

I somehow missed your PR and did the reST deprecations in PR #1403